index.js

```diff
@@ -1,8 +1,5 @@
 const { dialog, globalShortcut } = require('electron');
-
-function getShortcutFromConfig (key, config) {
-  return (config.hotkeys && config.hotkeys[key]) || config[`${key}Shortcut`];
-}
+import getShortcutFromConfig from './lib/index'
```

Comment:
Could we change this to a `require` to keep it consistent with the others (the line above and the test you added)?